src/FileHandler.php

```diff
@@ -172,4 +172,16 @@ public function renameFiles($manifest = null)
             rename($this->packagePath().'/'.$file, $this->packagePath().'/'.$filename);
         }
     }
+
+
+    public function cleanUpRules()
+    {
+        $ruleFiles = ['rules.php', 'rewriteRules.php'];
+
+        foreach($ruleFiles as $file) {
+            if(file_exists($this->packagePath(). '/' .$filename)) {
```

Comment:
As the tests point out, $filename does not exist.
Thanks so far for the PR though!

Reply:
haha whoops, seems like I forgot to rename the var after moving the code over